### DISCUSSION TITLE
General questions about this project

### DISCUSSION THREAD

**Discussion by anonymous-one** [thirtythreeforty/neolink#362] [General]
As usual, first and foremost, lovely project! Big thumbs up to all those involved!

Questions:

With this bridge, once everything is up and running, would one be able to block all WAN (internet) access by the camera and strictly use the rtsp bridge? On a related note, on reboot, do things tend to come back to a stable working state with internet access disabled?

I see that this project also publishes motion events via mqtt, with that in mind, since I would be using this with a battery (solar) powered camera, could the following be done (all scripted on my end of course):

a) motion triggered by camera, dispatched to mqtt
b) my script picks up this mqtt event, starts recording via rtsp for lets say 60 seconds
c) done

Thanks!

**Comment by QuantumEntangledAndy** [comment]
Firstly. Please do checkout my fork since this original one is mostly unmaintained.

Getting an mqtt connection to listen to motion events requires us to stay logged in. Which will drain the battery. There are plans to use push notifications https://github.com/QuantumEntangledAndy/neolink/pull/85 to get it to work instead which would mean we could stay logged out until such a notification but it's not implanted yet
